docs/agent_hocon_reference.md

```diff
@@ -632,3 +633,27 @@ Same as top-level [error_formatter above](#error_formatter), except at single-ag
 ### error_fragments
 
 Same as top-level [error_fragments above](#error_fragments), except at single-agent scope.
+
+### structure_formats
+
+_Front Man only_
+
+An optional list of strings describing the formats that the server-side should
+parse into the structure field of the ChatMessage response so clients do not have
+to re-invent this parsing wheel multiple times over.
+
+The first single structure found of the appropriate format(s) from the text of a response
+is what is put into the ChatMessage structure field, and any text which contributed to the
+parsing of that structure is removed from the ChatMessage text field.
+
+Supported values are:
+
+- "json"    Looks for JSON in the messages from the LLM and extracts
+
+Currently, the front-man is the only agent node that ever needs to specify this aspect of the [function](#function)
+definition.
+
+Example networks that parse their structure_formats:
+
+- [music_nerd_pro.hocon](../neuro_san/registries/music_nerd_pro.hocon)
```

Comment:
Add some docs about structure_formats.

FWIW: I have verified that it would be possible to include "structure_formats" within inner-agent function definitions and not screw any internals up. I did this by including a "structure_formats" definition within intranet_agents aaosa definition. This bodes well for using structure parsing in inter-agent communications in the future.